dynawo/sources/ModelicaCompiler/test/ModelicaModel/reference/GeneratorPQ_Dyn.cpp

```diff
@@ -679,54 +679,68 @@ void ModelGeneratorPQ_Dyn::defineVariables(std::vector<boost::shared_ptr<Variabl
 
 void ModelGeneratorPQ_Dyn::defineParameters(std::vector<ParameterModeler>& parameters)
 {
-  parameters.push_back(ParameterModeler("generator_AlphaPu", VAR_TYPE_DOUBLE, EXTERNAL_PARAMETER));
-  parameters.push_back(ParameterModeler("generator_PGen0Pu", VAR_TYPE_DOUBLE, EXTERNAL_PARAMETER));
-  parameters.push_back(ParameterModeler("generator_PMaxPu", VAR_TYPE_DOUBLE, EXTERNAL_PARAMETER));
-  parameters.push_back(ParameterModeler("generator_PMinPu", VAR_TYPE_DOUBLE, EXTERNAL_PARAMETER));
-  parameters.push_back(ParameterModeler("generator_QGen0Pu", VAR_TYPE_DOUBLE, EXTERNAL_PARAMETER));
-  parameters.push_back(ParameterModeler("generator_QMaxPu", VAR_TYPE_DOUBLE, EXTERNAL_PARAMETER));
-  parameters.push_back(ParameterModeler("generator_QMinPu", VAR_TYPE_DOUBLE, EXTERNAL_PARAMETER));
-  parameters.push_back(ParameterModeler("generator_U0Pu", VAR_TYPE_DOUBLE, EXTERNAL_PARAMETER));
-  parameters.push_back(ParameterModeler("generator_UMaxPu", VAR_TYPE_DOUBLE, EXTERNAL_PARAMETER));
-  parameters.push_back(ParameterModeler("generator_UMinPu", VAR_TYPE_DOUBLE, EXTERNAL_PARAMETER));
-  parameters.push_back(ParameterModeler("generator_i0Pu_im", VAR_TYPE_DOUBLE, EXTERNAL_PARAMETER));
-  parameters.push_back(ParameterModeler("generator_i0Pu_re", VAR_TYPE_DOUBLE, EXTERNAL_PARAMETER));
-  parameters.push_back(ParameterModeler("generator_u0Pu_im", VAR_TYPE_DOUBLE, EXTERNAL_PARAMETER));
-  parameters.push_back(ParameterModeler("generator_u0Pu_re", VAR_TYPE_DOUBLE, EXTERNAL_PARAMETER));
-  parameters.push_back(ParameterModeler("generator_NbSwitchOffSignals", VAR_TYPE_INT, SHARED_PARAMETER));
-  parameters.push_back(ParameterModeler("generator_State0", VAR_TYPE_INT, SHARED_PARAMETER));
+	using ParameterModelerTuple = std::tuple<std::string, DYN::typeVarC_t, DYN::parameterScope_t>;
```

Comment:
wrong indentation (should be 2, like before)

Reply:
The current indentation is 2 spaces

Reply:
the indentation is not the same as before